std_msgs/Header header
ml4kp_bridge/SpacePoint space_point
std_msgs/String space_name